static/js/blockly_blocks.js

```diff
@@ -46,4 +78,19 @@ export const initBlocks = () => {
   Blockly.JavaScript['input'] = (block) => {
     return ['inputArr[inputIdx++]', Blockly.JavaScript.ORDER_ATOMIC];
   };
+
+  Blockly.JavaScript['block_dropdown'] = function(block) {
+    var dropdown_drop_colour = block.getFieldValue('drop_colour');
+    let value = set_colour(dropdown_drop_colour);
+    return `
+            document.getElementById('console').innerText += ${value} + '\\n';
```

Comment:
Instead of using the block like this (setting the text as part of the block), it may make more sense to give this block an output and pass the value to the output block to print to console.

Reply:
removed it.